docs/docusaurus.config.js

```diff
@@ -0,0 +1,80 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'React Native Screens',
+  favicon: 'img/favicon.ico',
+
+  url: 'https://docs.swmansion.com',
+
+  baseUrl: '/',
+
+  organizationName: 'software-mansion',
+  projectName: 'react-native-screens',
+
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
```

Comment:
I'm wondering if we should `throw` an error, instead of warning if Docusaurus detects broken md links?